Fixtures/Miscellaneous/Plugins/MySourceGenPlugin/Plugins/MySourceGenBuildToolPlugin/plugin.swift

```diff
@@ -2,6 +2,11 @@ import PackagePlugin
 
 @main
 struct MyPlugin: BuildToolPlugin {
+    #if RETICULATE
```

Comment:
I find this term quite confusing. Where does it come from and what does it mean in this context?

Reply:
It doesn't mean anything: https://www.urbandictionary.com/define.php?term=reticulating%20splines

I needed a simple change that can be tested, so I decided to change the verb and add a joke.

Reply:
Do we want to keep that flag then in this PR and get it merged to `main`?

Reply:
imo this can stay, @MaxDesiatov do you have a concern with it? it is in a test fixure

Reply:
Sure, fine if it's in a fixture. I only feel that this could benefit from a clarifying comment in code for posterity saying that the name is arbitrary and doesn't have any special meaning.